xnb/objects/Texture2D.py

```diff
@@ -0,0 +1,89 @@
+
+from __future__ import annotations
+from typing import List
+from PIL import Image
+
+from ..constants import SurfaceFormat
+
+import logging
+import numpy
+import io
+
+class Texture2D:
+    def __init__(self) -> None:
```

Comment:
I feel like for a python "object", it should contain some kind of initialization parameters. I am also thinking of maybe using dataclasses here, but that's an optional thing.